### DISCUSSION TITLE
Allow to include shared photos on the map

### DISCUSSION THREAD

**Discussion by mvbjrn** [immich-app/immich#5019] [General]
As a user that really enjoys the map view, I would like to be able to also see photos on the map that got shared with me. Additionally, I would like to be able to configure this behavior, so that I can toogle different map views, e.g. depending on the user that shared photos with me.

**Comment by aisbergde** [comment]
Good idea, but not all assets, but only assets, I marked (for example, if I could mark them as favorites. Or better: a function "add (virtually) to my library"
